mlflow/langchain/__init__.py

```diff
@@ -271,6 +271,8 @@ def load_retriever(persist_directory):
     _validate_and_prepare_target_save_path(path)
 
     if isinstance(model_config, str):
+        if not os.path.isabs(model_config):
+            model_config = os.path.abspath(model_config)
         if os.path.exists(model_config):
             model_config = _load_from_yaml(model_config)
```

Comment:
once this code is removed, if no one is using `_load_from_yaml` utils. Lets delete the definition as well :)